.gitmodules

```diff
@@ -1,4 +1,6 @@
-[submodule "vendor/cpp-driver"]
-	path = vendor/cpp-driver
+[submodule "vendor/datastax-cpp-driver"]
+	path = vendor/datastax-cpp-driver
+	url = https://github.com/datastax/cpp-driver.git
```

Comment:
I guess that we should remove the no longer used CPP driver submodule?

Reply:
I kept both drivers to have build both for test performance etc..